public/scripts/raneto.js

```diff
@@ -18,8 +18,7 @@
       $(".content table").addClass("table");
 
       // FitVids
-      $(".content").fitVids();
-
+      fitVids();
```

Comment:
Was this not working as expected?
Looking only at this code, it seems like we would need a jQuery item to attach to instead of just initializing.